src/plugins/intel_gna/src/gna_plugin_config.cpp

```diff
@@ -46,7 +46,56 @@ static const caseless_unordered_map<std::string, std::pair<Gna2AccelerationMode,
 };
 OPENVINO_SUPPRESS_DEPRECATED_END
 
+/**
+ * @brief Parse string into value without throwing
+ * @param value A parsed value
+ * @param def_value A value to return in case of exception
+ * @return parsed value, default value otherwise
+ * @ingroup ov_runtime_gna_prop_cpp_api
+ */
+template <class T>
+static T FromString(const std::string& value, T def_value) {
+    T result = def_value;
+    try {
+        result = ov::util::from_string<T>(value);
+    } catch (...) {
```

Comment:
1. why do you need a wrapper around a single function?
2. `catch(...)` is not good practice because you ignore real user errors

Reply:
1. This function not only catches eventual exceptions, but also sets the default value to the returned value if it occurred. I also want to avoid try...catch in every place, where I use it, to improve code readability.
2. Changed to std::exception.